ansible_collections/arista/avd/roles/eos_l3ls_evpn/templates/fabric/mlag/leaf-mlag-spanning-tree.j2

```diff
@@ -1,4 +0,0 @@
-{# Leaf mlag disable spanning tree #}
```

Comment:
Why to remove this file completely ?

Reply:
I think I had an old version with unchanged things, re-added but not required for this PR... Maybe we can ignore?